js/patches/react-native+0.63.4.patch

```diff
@@ -1,30 +0,0 @@
-diff --git a/node_modules/react-native/react.gradle b/node_modules/react-native/react.gradle
-index c28cbd4..3b4bc55 100644
---- a/node_modules/react-native/react.gradle
-+++ b/node_modules/react-native/react.gradle
-@@ -305,6 +305,7 @@ afterEvaluate {
-         // mergeResources task runs before the bundle file is copied to the intermediate asset directory from Android plugin 4.1+.
-         // This ensures to copy the bundle file before mergeResources task starts
-         def mergeResourcesTask = tasks.findByName("merge${targetName}Resources")
-+        currentAssetsCopyTask.dependsOn(currentBundleTask) // See: https://github.com/facebook/react-native/issues/31157
-         mergeResourcesTask.dependsOn(currentAssetsCopyTask)
- 
-         packageTask.dependsOn(currentAssetsCopyTask)
-diff --git a/node_modules/react-native/scripts/ios-configure-glog.sh b/node_modules/react-native/scripts/ios-configure-glog.sh
-index 73a892c..de3e9cf 100755
---- a/node_modules/react-native/scripts/ios-configure-glog.sh
-+++ b/node_modules/react-native/scripts/ios-configure-glog.sh
-@@ -28,6 +28,13 @@ if [ -h "test-driver" ]; then
-     rm test-driver
- fi
- 
-+## @NOTE(gfanton): patch config.sub to work with apple silicon
-+# download patched config.sub for darwin arm64
-+curl -sSL --fail -o config.sub 'https://gist.githubusercontent.com/gfanton/f7dd19c299bef99d323cb08c650bb02e/raw/2f5a76ffd38cdac678adf6a9f12600e9d16e7345/config.darwin-arm-patch.sub'
-+
-+# check shasum of the file
-+echo 'a743811464b00f3e2372f5368035fdc38f322a3a  config.sub' | shasum -a 1 -c || exit 1
-+
- ./configure --host arm-apple-darwin
- 
- # Fix build for tvOS
```

Comment:
Are you sure we don't need this patch from @gfanton?
Did you try to compile the iOS app from XCode on both debug and release from a x86 and a M1 Mac?

Reply:
Yes, this patch was for recognize M1 like a valid arch (for flipper that use an old version of glog) but now it's detect automatically, so the builds (x86 and arm64) works 👍